### PR TITLE
Minor updates to the DiscreteTime class documentation.

### PR DESCRIPTION
**Pull request by bangerth**
Follow-up to #9748.

### CODE REVIEW

File: include/deal.II/base/discrete_time.h [hunk 1]
```diff
@@ -103,11 +114,12 @@ DEAL_II_NAMESPACE_OPEN
...
  *   commands within the user's code. Additionally, post-processed outputs can
- *   be prepared here which can be viewed later by visualization programs such
- *   as `Tecplot`, `Paraview`, and `VisIt`. Additionally, during the snapshot
- *   stage, the code can assess the quality of the previous step and decide
- *   whether it wants to increase or decrease the time step size. The step
- *   size for the next time step can be modified here.
+ *   be prepared here, which can then later be viewed by visualization programs
+ *   such as `Tecplot`, `Paraview`, and `VisIt`. Additionally, during the
+ * snapshot stage, the code can assess the quality of the previous step and
+ * decide whether it wants to increase or decrease the time step size. The step
```
Comment (kronbichler) [inline include/deal.II/base/discrete_time.h:120]: For consistency, do you want to indent these two lines with the previous ones, i.e., add two more spaces?

Comment (bangerth) [reply include/deal.II/base/discrete_time.h]: Yes, done. An effect of having too long a line that clang-format broke and I didn't check.

**Comment by bangerth** [comment]
@rezarastak -- FYI

File: include/deal.II/base/discrete_time.h [hunk 2]
```diff
@@ -103,11 +114,12 @@ DEAL_II_NAMESPACE_OPEN
...
- *   as `Tecplot`, `Paraview`, and `VisIt`. Additionally, during the snapshot
- *   stage, the code can assess the quality of the previous step and decide
- *   whether it wants to increase or decrease the time step size. The step
- *   size for the next time step can be modified here.
+ *   be prepared here, which can then later be viewed by visualization programs
+ *   such as `Tecplot`, `Paraview`, and `VisIt`. Additionally, during the
+ *   snapshot stage, the code can assess the quality of the previous step and
+ *   decide whether it wants to increase or decrease the time step size. The
+ *   step size for the next time step can be modified here, by calling
+ *   DiscreteTime::set_desired_next_step_size().
```
Comment (rezarastak) [inline include/deal.II/base/discrete_time.h:122]:
```suggestion
 *   set_desired_next_step_size().
```
Name of the class is not needed here and we don't have the class name prefix when referring to other methods. We can remove the prefix but It is totally fine if we keep it.

**Comment by bangerth** [comment]
@rezarastak -- thanks! I fixed that as well.

@dealii/dealii -- should be good to go!

/rebuild

**Comment by bangerth** [comment]
Ping?

**Pull request merged**